### PR TITLE
Add support for webp images

### PR DESCRIPTION
**Pull request by valib**
This is the first part of the implementation and this image format is not ignored and shown. But I have a problem to implement the https://github.com/sejda-pdf/webp-imageio to read those images. It raised an exception when instantiate the class.

EDIT: finally solved

### CODE REVIEW

**Comment by valib** [comment]
Now it is working

File: src/main/java/net/pms/dlna/LibMediaInfoParser.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import net.pms.formats.Format;
 import net.pms.formats.v2.SubtitleType;
 import net.pms.image.ImageFormat;
+import net.pms.image.ImageInfo;
```
Comment (SubJunk) [inline src/main/java/net/pms/dlna/LibMediaInfoParser.java]: This seems unused?

File: src/main/java/net/pms/image/ImagesUtil.java [hunk 2]
```diff
@@ -1333,7 +1333,7 @@ protected static Image transcodeImage(
 			throw new UnknownFormatException("Unable to read image format", e);
 		}
 
-		if (inputResult.bufferedImage == null || inputResult.imageFormat == null) { // ImageIO doesn't support the image format
+		if (inputResult.bufferedImage == null) { // ImageIO doesn't support the image format
```
Comment (SubJunk) [inline src/main/java/net/pms/image/ImagesUtil.java]: Why don't we need to check for that?

Comment (valib) [reply src/main/java/net/pms/image/ImagesUtil.java]: @SubJunk this was only for the testing. I returned it back in the final version

File: src/main/java/net/pms/dlna/LibMediaInfoParser.java (hunk 1 above)
Comment (valib) [reply src/main/java/net/pms/dlna/LibMediaInfoParser.java]: @SubJunk please check the final version

**Comment by valib** [comment]
@SubJunk during testing I have found that the latest implementation of the WIN10 Edge browser is not properly recognized and the icon is obsolete so I fix it.

**Comment by valib** [comment]
@SubJunk I think that we should make the correct description of the new classes or methods to keep the code understandable. This is the main problem of the current code. So we should add the description for the old code whenever is possible.

**Review by SubJunk — APPROVED** [review]

**Comment by SubJunk** [comment]
Nice work

**Pull request merged**